test/integration/tests/allow-newer-specific-packages/files/package.yaml

```diff
@@ -0,0 +1,8 @@
+name: foo
+version: 0
+
+dependencies:
+  - base >= 4.14 && < 4.15
```

Comment:
The YAML linter is opinionated when it comes to indentation. Don't indent this, and it will pass.

Reply:
Fixed both of these, thank you